go/hyperkit.go

```diff
@@ -108,9 +110,11 @@ type HyperKit struct {
 
 	// VSock enables the virtio-socket device and exposes it on the host.
 	VSock bool `json:"vsock"`
+	// VSockDir specifies where the VSock will be.  Stands for StateDir if empty.
```

Comment:
maybe "VSockDir specifies where the unix domain sockets will be created. Defaults to StateDir when empty."

Reply:
@rn Would you prefer that I rewrite the commits, or that I add fixes as additional commits?